.travis.yml

```diff
@@ -57,7 +57,11 @@ script:
   fi
 - bundle exec rake puppetserver
 after_script:
-- docker-compose -f spec/docker-compose.yml logs
+- |
+  sync
+  export line_sep=$(printf '%*s' 80 | tr ' ' '#')
+  docker ps --all -q | xargs -L 1 -I % sh -c 'echo "\n${line_sep}\nContainer Logs - $(docker inspect % --format={{.Name}})\n${line_sep}\n"; docker logs %'
```

Comment:
The downside is no colorized output... the upside is that the logs are no longer streamed together, hence readable!

Reply:
Do we plan to leave these in, or was it just for debugging this PR?

Reply:
It looks like it only takes 3.6 seconds, so that seems fine to leave in as long as it doesn't fail and cause travis to fail.